registry/storage/paths.go

```diff
@@ -18,6 +18,11 @@ const (
 	// basis.
 )
 
+var (
+	rootPrefix = []string{storagePathRoot, storagePathVersion}
+	repoPrefix = append(rootPrefix, "repositories")
+)
```

Comment:
You may want to put it back as local variables.

Think about the following code.
```go
storagePathVersion := "v2"
storagePathRoot := "/docker/registry/"
rootPrefix := []string{storagePathRoot, storagePathVersion}
repoPrefix := append(rootPrefix, "repositories")

pathA := append(repoPrefix, "foo")
pathB := append(repoPrefix, "bar")
fmt.Println(pathA)
fmt.Println(pathB)
```
The output is
```
[/docker/registry/ v2 repositories bar]
[/docker/registry/ v2 repositories bar]
```

Reply:
Put it back.